dm/templates/instance/instance.py.schema

```diff
@@ -25,53 +25,61 @@ required:
   - zone
   - machineType
   - diskImage
-  - network
```

Comment:
Please update your implementation in a way that both 'network' (as the current implementation) as 'networks' (the new implementation) is supported to assure backwards compatibility.